app/scripts/modules/core/delivery/executionGroup/execution/Execution.tsx

```diff
@@ -253,9 +253,12 @@ export class Execution extends React.Component<IExecutionProps, IExecutionState>
     const executionMarkerWidth = `${100 / this.props.execution.stageSummaries.length}%`;
     const executionMarkers = this.props.execution.stageSummaries.map((stage) => (
       <ExecutionMarker key={stage.refId}
+                       application={this.props.application}
+                       execution={this.props.execution}
                        stage={stage}
                        onClick={this.toggleDetails}
                        active={this.isActive(stage.index)}
+                       previousStageActive={this.isActive(stage.index - 1)}
```

Comment:
We were doing some CSS business to tweak the execution bar borders, but it was relying on `+` selectors, which breaks down with the changes to `ExecutionMarker`. This is sort of better, anyway?